codec/src/main/java/io/netty/handler/codec/DefaultTextHeaders.java

```diff
@@ -36,7 +36,7 @@ public int generateHashCode(CharSequence name) {
         }
     };
 
-    private static final HashCodeGenerator<CharSequence> CHARSEQUECE_CASE_SENSITIVE_HASH_CODE_GENERATOR =
+    private static final HashCodeGenerator<CharSequence> CHARSEQUENCE_CASE_SENSITIVE_HASH_CODE_GENERATOR =
```

Comment:
It's probably better dropping `CHARSEQUENCE_` at all? It's private and we all know it's for `CharSequence`.